Formula/ocaml-findlib.rb

```diff
@@ -0,0 +1,25 @@
+class OcamlFindlib < Formula
+  desc "OCaml library manager"
+  homepage "http://projects.camlcity.org/projects/findlib.html"
+  url "http://download.camlcity.org/download/findlib-1.7.3.tar.gz"
+  sha256 "d196608fa23c36c2aace27d5ef124a815132a5fcea668d41fa7d6c1ca246bd8b"
+
+  depends_on "ocaml"
+
+  def install
+    ENV.deparallelize
+
+    system "./configure", "-bindir", "#{HOMEBREW_PREFIX}/bin",
+                          "-mandir", "#{HOMEBREW_PREFIX}/share/man",
+                          "-sitelib", "#{HOMEBREW_PREFIX}/lib/ocaml",
+                          "-config", "#{HOMEBREW_PREFIX}/etc/ocamlfind.conf"
+    system "make", "all"
+    system "make", "opt"
+    system "make", "install", "prefix=#{buildpath}/output/"
```

Comment:
This seems weird to me. I think either upstream's prefix variable is broken (`prefix=#{prefix}` should work) or something is wrong with the options you're passing to configure. Maybe it expects relative paths?

Reply:
I tried a [version with relative paths](https://gist.github.com/btj/331045f679d55283422bc8482fa06366). It builds and installs fine (after I add the `-no-topfind` option), but the resulting `ocamlfind` tool does not work: it finds `ocamlfind.conf` only if launched from `/usr/local`, so obviously it expects the `-config` path to be absolute.

Notice that the [Debian](https://sources.debian.net/src/findlib/1.7.3-1/debian/rules/) and [Arch Linux](https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/ocaml-findlib) packages also use absolute path names.

Reply:
(I didn't check the Ubuntu, Cygwin, or MSYS2 packages, but I suspect they do, too.)

Reply:
OK, then there's a bug with `make prefix=#{prefix} install` not working that needs to be fixed upstream.

Reply:
Do I understand correctly that you feel that the approach I'm proposing is too ugly, and that upstream should take a patch so that this formula can be written in a cleaner way?

Reply:
Correct. It should not be necessary to stage things in `#{buildpath}/output` as `make install prefix=#{prefix}` should just work.